backend/tournesol/tests/test_api_comparison.py

```diff
@@ -763,3 +763,13 @@ def test_invalid_criteria_in_comparison(self):
         self.assertContains(
             response, "not a valid criteria", status_code=status.HTTP_400_BAD_REQUEST
         )
+
+    def test_missing_non_optional_criteria_in_comparison(self):
```

Comment:
How about moving this test near to the other tests using the same POST view? 

Let's say just after the test `test_authenticated_cant_create_criteria_scores_without_mandatory`?